recipe/meta.yaml

```diff
@@ -44,17 +43,19 @@ outputs:
       host:
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
         - cuda-version {{ cuda_version }}
+        - {{ pin_subpackage("cuda-crt-tools", exact=True) }}
+        - {{ pin_subpackage("cuda-nvvm-tools", exact=True) }}
```

Comment:
Why do we need these in host? I would have expected them to only be run requirements, especially since they only contain binaries.

Reply:
Let's try dropping these and see what happens

```suggestion
```

Reply:
The `files` constrain above is grabbing `bin/*`. It may make sense to include these just to avoid repackaging duplicate files that either of these packages would already have

Admittedly `bin/*` may be too sweeping. Though it does ensure that we have gotten everything else that was in `bin` that is not included in any other package